concepticondata/conceptlists/Greenhill-2015-2525.tsv

```diff
@@ -549,7 +549,7 @@ Greenhill-2015-2525-1269	1269	swollen
 Greenhill-2015-2525-1270	1270	to be thirsty	847	THIRSTY
 Greenhill-2015-2525-1271	1271	to be tired	1757	TIRED
 Greenhill-2015-2525-1272	1272	toenail	2662	TOENAIL
-Greenhill-2015-2525-1273	1273	tomorrow morning		
+Greenhill-2015-2525-1273	1273	tomorrow morning	1339	MORNING
```

Comment:
Is this a specific concept that only refers to the morning of the next day?

Reply:
I think it's just for tomorrow, remove the mapping?

Reply:
Yes, then I'd remove the mapping.

Reply:
ok, done.